### ISSUE TITLE
Right boställe for the right guest

### ISSUE THREAD

**Issue by patrik-pwned** [noQ-sweden/noq#660]
**Story**
As a guest i need to know if the boställe is right for me so i can be more sure that the boställe will accept my initial request.

See more criteria 

**Acceptance Criteria (checklist)**
- [ ] Add necessary guest criteria to the "Mer information" page of each boställe
- [ ] Add necessary guest criteria to the page where guests fill out the booking form

**Other info:** 
See ticket for the basic criteria.

- [ ]  **Definition of Ready OK?**
Also called DoR, our generic process for checking if this is ready for start. See our Playbook here for details: https://docs.google.com/document/d/1ou6wr1sI9IJZaGGSMn_SJvFru5FJQKMDRZY16kBmx5Y/edit#heading=h.3rdcrjn

- [ ]  **Definition of Done OK?**
Also called DoD, our generic process for checking this is done in the right way. See our Playbook here for details: https://docs.google.com/document/d/1ou6wr1sI9IJZaGGSMn_SJvFru5FJQKMDRZY16kBmx5Y/edit#heading=h.23ckvvd

**Comment by AbubakerSalah** [comment]
@patrik-pwned  Could you provide more details on this ticket? We have had a lot of changes since this ticket was written.

**Comment by patrik-pwned** [comment]
Yes, you're right. There use to be a ticket with more information on the shelters, i specifically remember listing all of them in the body of the ticket...

Basically, if the user is in the overview page and sees all of the shelters row by row, there should be some "bubbles" or "tags"  next to the shelter name stating for example "Only Women". The same information should be available when clicking on that specifik shelter for the detailed view/page. The point of this ticket is to allow the user to visually determine the correct shelter before clicking on the detailed view/page. It would be too time-consuming to click on every single shelter for the "Only Women" info if you are a woman. 

Imagine looking for a hotel, you have a list of all hotels in the area and you can see some essential criteria or information so you can quickly scan/qualify the listing without having to click on every single hotel.

**Comment by AntonNoQ** [comment]
Since the data on the detailpage is static coded and not from the database, I cannot show the data on the home/list page. Would it not be better to make this work forst? To get the actual data from the database, seems kind of a good thing to make sure the data is actually the data from the boställe.

**Comment by patrik-pwned** [comment]
I suppose so, would it require a lot of time and effort? Or is it an easy fix?

**Comment by AntonNoQ** [comment]
Good question, im new with react so im not sure. At some point it has to be done.
